static/app/components/timeline/index.tsx

```diff
@@ -87,7 +72,8 @@ const Row = styled('div')`
   }
   &:last-child {
     margin-bottom: 0;
-    background: ${p => p.theme.background};
+    /* Show/hide connecting line from the last element of the timeline */
+    background: ${p => (p.showLastLine ? 'transparent' : p.theme.background)} !important;
```

Comment:
Is there a way to avoid this `!important`?

Reply:
Sure is, I actually refactored it to remove it and then forgot to do that, thanks!